.pipelines/stages/jobs/steps/utils/download-ort-build.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
+- name: ep
+  type: string
+  default: cpu
+- name: os
+  type: string
+  default: linux
+- name: arch
+  type: string
+  default: x64
+steps:
+
+
+- download: ort_package
+  displayName: 'Download Pipeline Artifact from Zip-Nuget-Java-Nodejs Packaging Pipeline'
+  ${{ if eq(parameters.ep, 'directml') }}:
+    artifact: 'drop-nuget-dml'
+  ${{ elseif eq(parameters.ep, 'cuda') }}:
+    artifact: 'onnxruntime-${{ parameters.os}}-${{ parameters.ep}}-${{ parameters.arch}}-cuda'
+  ${{ else }}:
+    artifact: 'onnxruntime-${{ parameters.os}}-${{ parameters.ep}}-${{ parameters.arch}}'
+
+- ${{ if eq(parameters.ep, 'directml') }}:
+  - task: ExtractFiles@1
+    inputs:
+      archiveFilePatterns: '$(Pipeline.Workspace)/ort_package/*/*.nupkg'
+      destinationFolder: '$(Build.Repository.LocalPath)/ort'
```

Comment:
You may put the dest folder in $(Agent.TempDirectory) so that we will know it will definitely be cleaned out after each run. 
Otherwise, please add some comments here to explain how you avoid that.  The "overwriteExistingFiles" settings is also a dangerous usage. We should report an error and abort the pipeline instead silently doing overwrite. Because we do not want to reuse anything from a previous run.

Reply:
This file is removed